### DISCUSSION TITLE
Copy ZIGBEE2MQTT data from existing alpine LXC to debian LXC

### DISCUSSION THREAD

**Discussion by dmshimself** [tteck/Proxmox#1884] [General]
I don't seem to be able to update the current LXC I have that is running zigbee2mqtt to the latest version.  So my mind turned to running it with the debian image as that also has an update script.

But I'm not sure if there is a way to copy all the zigbee2mqtt data from an alpine LXC to a new debian one and repairing all the devices would be a pain.

If anyone has found a way to do this, that would be great.  I've had a go myself, but the front end doesn't start up.

**Comment by tteck** [comment]
The availability of Alpine app versions might be slightly delayed, but they will eventually be updated through the APK package manager.
You can keep check here https://pkgs.alpinelinux.org/packages?name=zigbee2mqtt&branch=edge&repo=&arch=x86_64&maintainer=

**Comment by tteck** [comment]
Furthermore, in the frontend, you have the option to request a backup, and once it's finished, you can proceed to download that backup.
![Screenshot 2023-10-04 8 46 52 PM](https://github.com/tteck/Proxmox/assets/21342670/d318842d-9ab7-4fdd-9c0b-7beb432875cf)
![Screenshot 2023-10-04 8 50 20 PM](https://github.com/tteck/Proxmox/assets/21342670/2bc25304-27a9-4738-94a2-42117e29ae10)

**Comment by dmshimself** [comment]
I've tried that before, but get the following error.  I thought it might be better just to try and switch to the debian version.  
![Screenshot_2023-10-05_21-44-17](https://github.com/tteck/Proxmox/assets/6901481/8811cea5-935d-4347-8987-11b635e0e734)
